packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2377,7 +2377,7 @@ const schema: SchemaType<DbUser> = {
   },
   subforumPreferredLayout: {
     type: String,
-    allowedValues: subforumLayouts,
+    allowedValues: Array.from(subforumLayouts),
```

Comment:
Now fixed separately in this PR: https://github.com/ForumMagnum/ForumMagnum/pull/6570